### ISSUE TITLE
Notifications not working on release builds after upgrading from 9.7.1

### ISSUE THREAD

**Issue by EsteveAguilera** [MaikuB/flutter_local_notifications#2060]
After upgrading from version 9.7.1, notifications are working for debug builds, but they do not work on release builds. This issue affects both Android and iOS.

Initially, we upgraded to the latest available version 15.1.0. After encountering the issue, we downgraded to version 10.0.0, but the problem persisted, leading us to believe that we missed some crucial steps during the upgrade from version 9.7.1 to version 10.

We're using Flutter version 3.10.1 

If you need any more information, please don't hesitate to ask. We appreciate any guidance you can provide.

Thanks in advance!

**Comment by MaikuB** [comment]
I suspect you haven't checked/followed the updated setup steps as you upgraded. Please recheck as updated setup steps are called out as part of changelog and typically part of a point aorund a breaking change. 

Note the plugin has an example app that is kept up to date with new releases and you should find that release builds still work. If you still believe there is actually an issue then you'll need to provide a link to repo hosting a minimal app. 10.0 has been out for nearly a year now and there are many others in the community who have used it or newer versions so I would expect that if an issue like this existed that there'd had many reports on this

**Comment by daphneyong11** [comment]
Hi, I am having trouble with the release version too, which the icon is not showing and custom sound not playing.
However, when I run with the debug and profile version it is working well.
This is my minimal flutter app, https://github.com/daphneyong11/notification

**Comment by MaikuB** [comment]
@daphneyong11 this is because you hadn't gone through and done the release build configuration that is mentioned in the readme. Please take a look at the readme and cross reference with the example app

**Comment by EsteveAguilera** [comment]
Hi @MaikuB,

After going through the updated setup steps again and analyzing our setup, I discovered that the issue wasn't related to the plugin.

For Android, upgrading the Gradle version as mentioned in the documentation fixed the push notifications problem. For iOS, I discovered there is an issue with the Firebase Token on release build which was causing the issue, so I hope once we fix it the notifications will work again.

I apologize for any confusion and inconvenience. The problem was on our end and not a bug with the plugin. I'm going to close this issue now.

Thank you again for your support and for maintaining such a useful plugin.
